specification/app/data-plane/Microsoft.App.DynamicSessions/preview/2024-10-02-preview/DynamicSessions.json

```diff
@@ -0,0 +1,759 @@
+          }
+        ]
+      }
+    },
+    "PagedSessionResourceFile": {
+      "type": "object",
+      "description": "Paged collection of SessionResourceFile items",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The SessionResourceFile items on this page",
+          "items": {
+            "$ref": "#/definitions/SessionResourceFile"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "SessionCodeExecutionAsyncResponse": {
+      "type": "object",
+      "description": "The response of the async code execution.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Session code execution id.",
+          "readOnly": true
+        },
+        "status": {
+          "$ref": "#/definitions/CodeExecutionStatus",
+          "description": "The status of the code execution, indicates whether succeeded or not."
+        },
+        "stdout": {
+          "type": "string",
+          "description": "The standard output of the code execution."
+        },
+        "stderr": {
+          "type": "string",
+          "description": "The standard error of the code execution."
+        },
+        "result": {
+          "description": "The result of the code execution. The type of this field is same as the type of actual result of the code execution after being Json serialized."
+        },
+        "executionTimeInMilliseconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The execution time of the code in milliseconds."
+        }
+      },
+      "required": [
+        "id",
+        "status",
+        "stdout",
+        "stderr",
+        "result",
+        "executionTimeInMilliseconds"
+      ]
+    },
+    "SessionCodeExecutionRequest": {
+      "type": "object",
+      "description": "The request to execute code.",
+      "properties": {
+        "codeInputType": {
+          "$ref": "#/definitions/CodeInputType",
+          "description": "The code input type of the code execution request."
+        },
+        "executionType": {
+          "$ref": "#/definitions/ExecutionType",
+          "description": "The execution type of the code execution request."
+        },
+        "code": {
+          "type": "string",
+          "description": "The string of the code to execute, based on CodeInputType."
+        },
+        "timeoutInSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Code execution timeout in seconds."
+        }
+      },
+      "required": [
+        "codeInputType",
+        "executionType",
+        "code",
+        "timeoutInSeconds"
+      ]
+    },
+    "SessionCodeExecutionResponse": {
```

Comment:
This appears to be a status model for an LRO operation, and if that's the case it must include an "error" property with the standard error schema.

Reply:
Yes, sure, added.